docs/speech.rst

```diff
@@ -1,18 +1,6 @@
 Speech
 ******
 
-.. warning::
-
-    WARNING! THIS IS ALPHA CODE.
-
-    We reserve the right to change this API as development continues.
-
-    The quality of the speech is not great, merely "good enough". Given the
```

Comment:
Maybe worth keeping some of this information while still removing any mentions of this being under work.